_config.yml

```diff
@@ -10,28 +10,11 @@ title: This is the site title
 # See https://github.com/bryanbraun/anchorjs for more information.
 # anchor_js_targets: [h1, h2, h3, h4, h5, h6]
 
-# Configuration for jekyll_pages_api_search plugin gem.
-jekyll_pages_api_search:
-  # Uncomment this to speed up site generation while developing.
-  #skip_index: true
 
-  # Each member of `index_fields` should correspond to a field generated by
-  # the jekyll_pages_api. It can hold an optional `boost` member as a signal
-  # to Lunr.js to weight the field more highly (default is 1).
-  index_fields:
-    title:
-      boost: 10
-    tags:
-      boost: 10
-    url:
-      boost: 5
-    body:
-  results_page_title: Search results # Used for the title tag and the header on the results page
+# To enable search, uncomment the search section
+# You will need to setup a search account with search.gov
+# https://search.usa.gov/signup
+# site_handle - is from search.gov
 
-# Page defaults
-# This default setting is used in conjunction with the jekyll_pages_api_search to hide results from the USWDS from showing up in the search results
-defaults:
-  - scope:
-      path: "assets/uswds"
-    values:
-      skip_index: true
+#search:
```

Comment:
Since there is only one key under `search` I don't think it's necessary to have them nested like this. Maybe we could just flatten this out to `search_site_handle`? Do you expect there to be any other search config values being added down the line?

Reply:
Yeah that sounds good 👍